2021-05-19 15:22:37,223 - src - __main__ - <module> - 11 - DEBUG - Calling module function.
2021-05-19 15:22:37,223 - src - __main__ - <module> - 16 - ERROR - division by zero
Traceback (most recent call last):
  File "C:\Users\user\Desktop\ML_work\1fc1e6bc02e1c6ce7e5ac523fed9568f\src\__main__.py", line 13, in <module>
    module.multiply(5, 0)
  File "C:\Users\user\Desktop\ML_work\1fc1e6bc02e1c6ce7e5ac523fed9568f\src\module.py", line 8, in multiply
    return num1 / num2
ZeroDivisionError: division by zero
2021-05-19 15:30:53,985 - src - __main__ - <module> - 11 - DEBUG - Calling module function.
2021-05-19 15:30:53,987 - src - __main__ - <module> - 16 - ERROR - division by zero
Traceback (most recent call last):
  File "C:\Users\user\Desktop\ML_work\1fc1e6bc02e1c6ce7e5ac523fed9568f\src\__main__.py", line 13, in <module>
    module.multiply(5, 0)
  File "C:\Users\user\Desktop\ML_work\1fc1e6bc02e1c6ce7e5ac523fed9568f\src\module.py", line 8, in multiply
    return num1 / num2
ZeroDivisionError: division by zero
2021-05-19 15:31:35,165 - src - __main__ - <module> - 11 - DEBUG - Calling module function.
2021-05-19 15:31:35,166 - src - __main__ - <module> - 16 - ERROR - division by zero
Traceback (most recent call last):
  File "C:\Users\user\Desktop\ML_work\1fc1e6bc02e1c6ce7e5ac523fed9568f\src\__main__.py", line 13, in <module>
    module.multiply(5, 0)
  File "C:\Users\user\Desktop\ML_work\1fc1e6bc02e1c6ce7e5ac523fed9568f\src\module.py", line 8, in multiply
    return num1 / num2
ZeroDivisionError: division by zero
2021-05-19 16:08:33,766 - src - __main__ - <module> - 11 - DEBUG - Calling module function.
2021-05-19 16:08:33,767 - src - __main__ - <module> - 16 - ERROR - division by zero
Traceback (most recent call last):
  File "C:\Users\user\Desktop\ML_work\1fc1e6bc02e1c6ce7e5ac523fed9568f\src\__main__.py", line 13, in <module>
    module.multiply(5, 0)
  File "C:\Users\user\Desktop\ML_work\1fc1e6bc02e1c6ce7e5ac523fed9568f\src\module.py", line 8, in multiply
    return num1 / num2
ZeroDivisionError: division by zero
2021-05-19 16:24:07,110 - src - __main__ - <module> - 5 - INFO - ------------------------- NEW RUN --------------------------------------
2021-05-19 16:24:07,111 - src - __main__ - <module> - 7 - DEBUG - Calling module function.
2021-05-19 16:24:07,148 - src - __main__ - <module> - 12 - ERROR - division by zero
Traceback (most recent call last):
  File "C:\Users\user\Desktop\ML_work\1fc1e6bc02e1c6ce7e5ac523fed9568f\src\__main__.py", line 9, in <module>
    module.multiply(5, 0)
  File "C:\Users\user\Desktop\ML_work\1fc1e6bc02e1c6ce7e5ac523fed9568f\src\module.py", line 9, in multiply
    return num1 / num2
ZeroDivisionError: division by zero
2021-05-19 16:33:35,721 - src - __main__ - <module> - 5 - INFO - ------------------------- NEW RUN --------------------------------------
2021-05-19 16:33:35,722 - src - __main__ - <module> - 7 - DEBUG - Calling module function.
2021-05-19 16:33:35,722 - src.module - module - multiply - 8 - DEBUG - Executing multiply function.
2021-05-19 16:33:35,723 - src - __main__ - <module> - 12 - ERROR - division by zero
Traceback (most recent call last):
  File "C:\Users\user\Desktop\ML_work\1fc1e6bc02e1c6ce7e5ac523fed9568f\src\__main__.py", line 9, in <module>
    module.multiply(5, 0)
  File "C:\Users\user\Desktop\ML_work\1fc1e6bc02e1c6ce7e5ac523fed9568f\src\module.py", line 9, in multiply
    return num1 / num2
ZeroDivisionError: division by zero
2021-05-19 16:36:02,772 - src - __main__ - <module> - 5 - INFO - ------------------------- NEW RUN --------------------------------------
2021-05-19 16:36:02,772 - src - __main__ - <module> - 7 - DEBUG - Calling module function.
2021-05-19 16:36:02,772 - src - __main__ - <module> - 12 - ERROR - division by zero
Traceback (most recent call last):
  File "C:\Users\user\Desktop\ML_work\1fc1e6bc02e1c6ce7e5ac523fed9568f\src\__main__.py", line 9, in <module>
    module.multiply(5, 0)
  File "C:\Users\user\Desktop\ML_work\1fc1e6bc02e1c6ce7e5ac523fed9568f\src\module.py", line 9, in multiply
    return num1 / num2
ZeroDivisionError: division by zero
2021-05-19 16:42:20,020 - src - __main__ - <module> - 18 - INFO - ------------------------- NEW RUN --------------------------------------
2021-05-19 16:42:20,020 - src - __main__ - <module> - 20 - DEBUG - Calling module function.
2021-05-19 16:42:20,020 - src - __main__ - <module> - 25 - ERROR - division by zero
Traceback (most recent call last):
  File "C:\Users\user\Desktop\ML_work\1fc1e6bc02e1c6ce7e5ac523fed9568f\src\__main__.py", line 22, in <module>
    module.multiply(5, 0)
  File "C:\Users\user\Desktop\ML_work\1fc1e6bc02e1c6ce7e5ac523fed9568f\src\module.py", line 9, in multiply
    return num1 / num2
ZeroDivisionError: division by zero
